force-app/main/default/classes/GeocodingServiceTest.cls

```diff
@@ -29,18 +29,9 @@ private with sharing class GeocodingServiceTest {
         );
 
         // THEN
-        System.assert(
-            computedCoordinates.size() == 1,
-            'Expected 1 pair of coordinates were returned'
-        );
-        System.assert(
-            computedCoordinates[0].lat == LATITUDE,
-            'Expected mock lat was returned'
-        );
-        System.assert(
-            computedCoordinates[0].lon == LONGITUDE,
-            'Expected mock lon was returned'
-        );
+        Assert.areEqual(computedCoordinates.size(), 1);
```

Comment:
Param orders are incorrect for all `Assert.areEqual` in this file: it's expected value first then, actual.